src/testsuite/vec_pwr9_dummy.c

```diff
@@ -1174,6 +1182,7 @@ __test_vec_extract_sig_f32 (vf32_t val)
 }
 #endif
 
+#ifndef __clang__
```

Comment:
why? add a comment.

Reply:
// Clang does not support the PWR9 bcd builtins.

Reply:
// Clang does not support vec_bcd_ppc.h period